machine_configuration/mco-update-boot-images.adoc

```diff
@@ -16,9 +16,9 @@ This process could cause the following issues:
 * Certificate expiration issues
 * Version skew issues
 
-To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters and as a Technology Preview feature for Amazon Web Services (AWS) clusters. It is not supported for clusters managed by the {cluster-capi-operator}.
+To avoid these issues, you can configure your cluster to update the boot image whenever you update your cluster. By modifying the `MachineConfiguration` object, you can enable this feature. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) and Amazon Web Services (AWS) clusters and as a Technology Preview feature for VMware vSphere clusters. It is not supported for clusters managed by the {cluster-capi-operator}.
 
-:FeatureName: The updating boot image feature for AWS
+:FeatureName: The updating boot image feature for vSphere
```

Comment:
Same as above, we should probably drop this block, since we are not having any boot image updates feature in tech preview for 4.18